README.md

```diff
@@ -157,6 +157,9 @@ options:
   ignored_keywords:
     - DO NOT REVIEW
   enable_group_assignment: false
+
+  # Set 0 to add all the reviewers (default: 0)
```

Comment:
I think it would be more intuitive not to set any default value, and place no limit on the number of reviewers when this option isn't explicitly set. Then, the comment can be like that?

```suggestion
  # To add all the reviewers, do not set this option
```

Reply:
Resolved in #30.